server/tests/raft_metadata_cluster.rs

```diff
@@ -190,14 +190,24 @@ async fn raft_metadata_cluster_chaos_test() -> googletest::Result<()> {
             expected_recovery_duration: Duration,
         ) -> anyhow::Result<Infallible> {
             loop {
+                let metadata_cluster_size = cluster
+                    .nodes
+                    .iter()
+                    .filter(|node| node.config().has_role(Role::MetadataServer))
+                    .count();
                 let node = cluster
                     .nodes
                     .choose_mut(&mut rand::rng())
                     .expect("at least one node being present");
                 node.restart().await?;
                 success_rx.mark_unchanged();
                 cluster
-                    .wait_check_healthy(HealthCheck::MetadataServer, Duration::from_secs(10))
+                    .wait_check_healthy(
+                        HealthCheck::MetadataServer {
+                            metadata_cluster_size,
```

Comment:
Wondering if we should explicitly set this to 0 to preserve original test behaviour.

Reply:
Did you see the `raft_metadata_cluster_chaos_test` failing recently?